python/solver.py

```diff
@@ -178,8 +178,8 @@ def multiply_bloch_phase(self, arr):
         return np.reshape(arr, dims)
 
     def get_poynting(self, which_band):
-        e = self.get_efield(which_band).ravel()
-        h = self.get_hfield(which_band).ravel()
+        e = self.get_efield(which_band, False).ravel()
+        h = self.get_hfield(which_band, False).ravel()
```

Comment:
It actually doesn't matter if you multiply by the Bloch phase here or not, since it will end up cancelling in the Poynting vector.  But we might as well pass `False` here to avoid doing unnecessary work.